forum/settings.py

```diff
@@ -0,0 +1,144 @@
+"""
+Django settings for forum project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+# from decouple import config
+
+
+load_dotenv()
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-t&h(6bczgs&7s8to0qp5i)csn_izs%=t!*q(km=fhk4l@$p2xl'
```

Comment:
Please take the secret key
SECRET_KEY = os.environ.get('SECRET_KEY')

Reply:
Fixed